internal/server/auditsink/logfile/logfile.go

```diff
@@ -0,0 +1,57 @@
+package logfile
+
+import (
+	"encoding/json"
+	"os"
+	"sync"
+
+	"go.flipt.io/flipt/internal/server/auditsink"
+	"go.uber.org/zap"
+)
+
+const sinkType = "logfile"
+
+// LogFileSink is the structure in charge of sending Audits to a specified file location.
+type LogFileSink struct {
+	logger *zap.Logger
+	file   *os.File
+	mtx    sync.Mutex
+}
+
+// NewLogFileSink is the constructor for a LogFileSink.
+func NewLogFileSink(logger *zap.Logger, filePath string) (*LogFileSink, error) {
+	file, err := os.Create(filePath)
```

Comment:
I think we'll likely want to change this to create OR append, similar to how zap does it here: https://github.com/uber-go/zap/blob/v1.24.0/sink.go#L157

otherwise we will overwrite (truncate) an existing log file if present